config/database.yml

```diff
@@ -21,8 +21,19 @@ default: &default
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
 
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+sqlite3: &sqlite3
+  adapter: sqlite3
+  pool: 5
+  timeout: 5000
+
 development:
-  <<: *default
+  <<: *sqlite3
```

Comment:
I'm not sure if we should change this right now. Maybe revert this file to default and we can gather somemore info on whether we should be using sqlite3, especially since we have other dependencies on PostgreSQL

Reply:
reverted!